cloudpub/ms_azure/utils.py

```diff
@@ -21,6 +22,19 @@
 log = logging.getLogger(__name__)
 
 
+UNSUPPORTED_SECURITY_TYPE_ARCHES = (
+    os.environ.get("UNSUPPORTED_SECURITY_TYPE_ARCHES", "") or "x64Gen1"
+)
+"""
+The list of arches that don't support security type.
+
+This is a comma-separated list of arches that don't support security type.
+It's used to skip the security type for the arches that don't support it.
+It's set as an environment variable to allow for customization.
+If the environment variable is empty, the default "x64Gen1" is used.
```

Comment:
What does Azure marketplace documentation says about `skus, imageType, securityType  ` for the schema virtual-machine-plan-technical-configuration ?

I have a feeling there might be different expected values based on different settings of plan in web-ui.